extensions/arc/src/ui/components/radioOptionsGroup.ts

```diff
@@ -0,0 +1,76 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import * as azdata from 'azdata';
+import * as vscode from 'vscode';
+import { getErrorMessage } from '../../common/utils';
+
+export interface RadioOptionsInfo {
+	values?: string[],
+	defaultValue: string
+}
+
+export class RadioOptionsGroup {
+	static id: number = 1;
+	private _divContainer!: azdata.DivContainer;
+	private _loadingBuilder: azdata.LoadingComponentBuilder;
+	private _currentRadioOption!: azdata.RadioButtonComponent;
+
+	constructor(private _view: azdata.ModelView, private _onNewDisposableCreated: (disposable: vscode.Disposable) => void, private _groupName: string = `RadioOptionsGroup${RadioOptionsGroup.id++}`) {
+		const divBuilder = this._view.modelBuilder.divContainer();
+		const divBuilderWithProperties = divBuilder.withProperties<azdata.DivContainerProperties>({ clickable: false });
+		this._divContainer = divBuilderWithProperties.component();
+		const loadingComponentBuilder = this._view.modelBuilder.loadingComponent();
+		this._loadingBuilder = loadingComponentBuilder.withItem(this._divContainer);
+	}
+
+	public component(): azdata.LoadingComponent {
```

Comment:
you meant to make this a getter, right? 

```
public get component(): azdata.LoadingComponent
```

Reply:
Just trying to keep usage consistent with azdata.component methods. It makes for easier usage.